components/menu/demo/menu-item.html

```diff
@@ -28,6 +29,12 @@ <h2>Menu Item (link)</h2>
 				<d2l-menu-item-link href="https://en.wikipedia.org/wiki/Universe" text="The Universe (link)"></d2l-menu-item-link>
 			</d2l-demo-snippet>
 
+			<h2>Menu Item (return)</h2>
+
+			<d2l-demo-snippet>
+				<d2l-menu-item-return text="Solar System"></d2l-menu-item-return>
```

Comment:
I imagine once we have the menu as well, we'll have a collective demo, since these menu item components should not be used outside of a menu.

Reply:
For sure. I also plan on making better tests/adding visual diff tests once menu exists. This demo is more of a for now thing to be easily able to see what exists for this component.